webui/react/src/services/api.ts

```diff
@@ -45,6 +46,21 @@ const userApi:  Api<{}, User> = {
 
 export const getCurrentUser = generateApi<{}, User>(userApi);
 
+export interface ExperimentsParams {
+  states?: string[];
+}
+
+const experimentsApi:  Api<ExperimentsParams, Experiment[]> = {
+  httpOptions: (params) => ({
+    url: '/experiment-summaries' + (params.states ? '?'+params.states.join(',') : ''),
```

Comment:
I might be missing something here but is this missing a `states=` prefix?

also suggestion:
```
url: `/experiment-summaries?states=${(params.states || []).join(',')}`,
```

Reply:
good catch. this is an issue.